releasenotes/notes/fixed-bug-in-Optimize1qGatesDecomposition-skipping-short-sequences-044a64740bf414a7.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
This file you have to delete all the boilerplate and keep only the part that is relevant to this PR.

Reply:
haha, serves me right for not looking at the diff